portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,8 +43,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String comment = getComment(request);
 
         // Add comment to comments Array.
-        comments.add(comment);
-
+        if(comment != null){
+            comments.add(comment);
+        }
```

Comment:
tiny nit: space after if and the closing parenthesis

`if (comment != null) {`

This looks great. In the later exercises (not in this PR) let's revisit this and see what improvements we can make to error handling.

Reply:
Ack